File: Data_Structure/Linked_List/1.LinkedList.java
public class LinkList {
  Node head; // head of the list

  static class Node {
    int data;
    Node next;

    Node(int d) {
      data = d;
      next = null;
    } // constructor
  }

  public void printList() {
    Node n = head;
    while (n != null) {
      System.out.print(n.data + " ");
      n = n.next;
    }
  }

  public static void main(String[] args) {
    LinkList ll = new LinkList();
    ll.head = new Node(1);
    Node second = new Node(2);
    Node third = new Node(3);

    ll.head.next = second;
    second.next = third;
    third.next = null;
    ll.printList();
  }
}

File: byte-int-prep/1.LinkedList.cpp
#include <iostream>
using namespace std;

struct Node
{
	int data;
	Node *next;
};

Node *head;

void PrintNode()
{
	Node *temp = head;
	cout << "List is: " << endl;
	while (temp)
	{
		cout << temp->data << " ";
		temp = temp->next;
	}
	cout << endl;
}

void Insert(int x)
{
	Node *temp = new Node();
	temp->data = x;
	temp->next = head;
	head = temp;
}

int main()
{
	cout << "How Many Number: " << endl;
	int n;
	cin >> n;
	for (int i = 0; i < n; i++)
	{
		cout << "Enter Number: " << endl;
		int x;
		cin >> x;
		Insert(x);
	}
	PrintNode();
	cout << endl;
}